forward_engineering/helpers/updateHelper.js

```diff
@@ -387,6 +395,9 @@ const columns = {
 }
 
 const generateScript = (child, udtMap, data, column, mode) => {
```

Comment:
It is preferrable to name functions like <do-something-**_Script_**> if the return a script - i.e. a string. If it doesn't you should name it like <do-something-**_ScriptDto_**>

Reply:
You should've also included JsDocs to refactored and added functions